build.sbt

```diff
@@ -127,9 +126,8 @@ lazy val json4s = project
   .settings(
     libraryDependencies ++= Seq(
       "com.softwaremill.sttp.client3" %% "json4s" % sttpClientVersion,
-      "org.json4s" %% "json4s-jackson" % json4sVersion
-    ),
-    crossScalaVersions := crossScala2Versions
```

Comment:
Overriding here with `crossScalaVersions := Seq(scala2_12, scala2_13)` should do the trick, since json4s doesn't support scala 3 yet.

Reply:
I tried this in my previous PR, but it didn't help (failed in publish section). Wonder why 🤔

Reply:
Probably, i will temporarily remove spray-json & json4s modules from build.sbt to be able to publish at least circe

Reply:
@nryanov can we try my fix #297 ?  
 Build needs approval..

Reply:
Yep, sure

I've approved build